src/openms/source/FORMAT/DATAACCESS/SiriusFragmentAnnotation.cpp

```diff
@@ -91,9 +91,10 @@ namespace OpenMS
     }
 
     // convert to vector
+    annotated_spectra.reserve(native_ids_annotated_spectra.size());
     for (const auto& it : native_ids_annotated_spectra)
     {
-      annotated_spectra.emplace_back(std::move(it.second));
+      annotated_spectra.emplace_back(it.second);
```

Comment:
@timosachsenberg @oliveralka this seems like it makes things worse?

Reply:
Move of const ref

Reply:
looking at this code, it seems that the const ref should not be const? @oliveralka

Reply:
will add to #5685